.github/actions/run-backend-tests/action.yml

```diff
@@ -78,6 +78,12 @@ runs:
               node-version: 18.12.1
               cache: pnpm
 
+        # tests would intermittently fail in GH actions
+        # with exit code 134 _after passing_ all tests
+        # this appears to fix it
+        # absolute wild tbh https://stackoverflow.com/a/75503402
+        - uses: tlambert03/setup-qt-libs@v1
```

Comment:
I'm so confused this worked, given we don't use pytest-qt at all, which all the more impresses me that you managed to figure this out 🙈

Reply:
it worked once 🤷 
let's roll the dice and see what happens